.env

```diff
@@ -1,2 +1,3 @@
 VUE_APP_ROOT_API=http://localhost:8181/api/v1
+VUE_APP_ROOT_ADMIN_API=http://localhost:8282/api/v1
```

Comment:
Sai rồi anh, không móc tự tiếp từ Admin qua
Ở BE user dùng OpenFeign để gọi tới. Chứ không phải gọi api của admin trực tiếp từ user đâu